cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,64 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
+import SettingsPage from '../support/pages/settings.pageObject';
```

Comment:
Please add fulfil this requirement:

Record a video of your running your tests (you can use Loom).
Attach a link to your video to the PR description.

Current link does not work